src/App.js

```diff
@@ -1,14 +1,78 @@
-import React from 'react';
+import React from 'react'
+import TodoForm from './components/TodoComponents/TodoForm'
+import TodoList from './components/TodoComponents/TodoList'
+import './components/TodoComponents/Todo.css'
+
+
+localStorage.getItem('state')
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props){
+    super(props);
+    this.state =
+      {Todos: 
+        [
+          {task: 'watch this week\'s The Orville',
+          id: 1528817077286,
+          completed: false},
+          {task: 'watch this week\'s Star Trek Discovery',
+          id: 1528817071031,
+          completed: false},
+        ],
+        Todo: '',
+      }
+  }
+
+  newTodo = (event) => this.setState({Todo: event.target.value})
+
+  addTask = (event) => {
```

Comment:
this is great but make sure you're adding comments in your code, treat this as if it was a group project or even something you might pick up later down the line and want to look at. Looking at old code or someone elses code can be a challenge that you can simplify with some comments.